protocols/0f30a4/sample_aliquoting.ot2.py

```diff
@@ -0,0 +1,106 @@
+from opentrons import labware, instruments
+from otcustomizers import StringSelection
+import math
+
+metadata = {
+    'protocolName': 'Sample Aliquoting',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# create custom labware
+plate_name = 'agilent_24_wellplate_10ml'
+if plate_name not in labware.list():
+    labware.create(
+        plate_name,
+        grid=(6, 4),
+        spacing=(18, 18),
+        diameter=17.2,
+        depth=40.3,
+        volume=10000
+    )
+
+# load labware
+plate = labware.load(plate_name, '1', 'plate')
+tuberack50 = labware.load('opentrons_6_tuberack_falcon_50ml_conical', '2')
+tuberack15 = labware.load(
+    'opentrons_15_tuberack_falcon_15ml_conical', '3')
+res12 = labware.load('usascientific_12_reservoir_22ml', '5')
+tips1000 = labware.load('opentrons_96_tiprack_1000ul', '7')
+tips300 = labware.load('opentrons_96_tiprack_300ul', '8')
+
+
+def run_custom_protocol(
+        p1000_mount: StringSelection('right', 'left') = 'right',
+        p300_mount: StringSelection('left', 'right') = 'left'
+):
+    # check
+    if p1000_mount == p300_mount:
+        raise Exception('Input different mounts for P1000 and P300 pipettes.')
+
+    # pipettes
+    p1000 = instruments.P1000_Single(mount=p1000_mount, tip_racks=[tips1000])
+    p1000.start_at_tip(tips1000.wells('B1'))
+    p300 = instruments.P300_Single(mount=p300_mount, tip_racks=[tips300])
+    tips1000.wells('C1')
```

Comment:
What does this line do?

Reply:
should be `p300.start_at_tip(tips300.wells('C1')`, sorry!